.devcontainer/devcontainer.json

```
@@ -9,8 +9,13 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/universal
 {
+    // Use the image from the CI
     "image": "ghcr.io/gpuweb/gpuweb:main",
-
+    // Build the image from the Dockerfile
+    // "build": {
+    //     "dockerfile": "../tools/custom-action/Dockerfile",
+    //     "context": ".."
+    // },
```

This is an alternative to "image", right? This makes it slightly clearer:
```suggestion
    // Use this instead if you need to build the image from the local Dockerfile:
    // "build": {
    //     "dockerfile": "../tools/custom-action/Dockerfile",
    //     "context": ".."
    // },

```

---

@kainino0x yes indeed! Thank you.